mockOSApi.v2/Controllers/ProcessController.cs

```diff
@@ -1,31 +1,56 @@
 using mockOSApi.Services;
 using Microsoft.AspNetCore.Mvc;
 using mockOSApi.DTO;
-using Microsoft.AspNetCore.Authorization;
-
+using Microsoft.Extensions.Localization;
 
+/// <summary>
+/// Controller responsible with Process handling
+/// </summary>
 [ApiController]
 [Route("/api/[controller]")]
 [Produces("application/json")]
 public class ProcessController : Controller
 {
     private readonly ILogger<ProcessController> _logger;
+
+    private readonly IStringLocalizer<ErrorMessage> _localizer;
     private readonly IProcessService _processService;
 
-    public ProcessController(ILogger<ProcessController> logger, IProcessService processHandler)
+    private readonly IAuthentication _authorizationService;
+
+    public ProcessController(ILogger<ProcessController> logger, IProcessService processHandler, IAuthentication authorizationService,IStringLocalizer<ErrorMessage> localizer)
     {
         _logger = logger;
         _processService = processHandler;
+        _authorizationService = authorizationService;
+        _localizer = localizer;
+    }
 
+    private string? FetchTokenFromHeaders()
```

Comment:
If you will use this in multiple places you should create a new Controller base class